pkg/paho-mqtt/Makefile.include

```diff
@@ -0,0 +1,18 @@
+INCLUDES += -I$(PKGDIRBASE)/paho-mqtt/MQTTClient-C/src/
+INCLUDES += -I$(PKGDIRBASE)/paho-mqtt/MQTTPacket/src
+INCLUDES += -I$(RIOTBASE)/pkg/paho-mqtt/include
+
+DIRS += $(RIOTBASE)/pkg/paho-mqtt/contrib
+
+CFLAGS += -DSOCK_HAS_IPV6
+CFLAGS += -DSOCK_HAS_IPV4
```

Comment:
Sometimes the simplest answer is the right one. @gschorcht @javierfileiv I think this is why `SOCK_HAS_IPV6` is always defined and the address always has the size of an IPv6 address. @javierfileiv please remove these two lines. The definition of those macros should be left to the corresponding `sock` modules.

Reply:
Ups... :$

Reply:
Shall I remove those LWIP specific lines below? line 7 and 8?

Reply:
Yes, please. Also, please add a comment, why those changes to the configuration are required.

Reply:
To be honest, I don't know why I did put those... I just played with lwip example, and try it to make it work. I know it's not so pro, but it worked. I put some comment though. Could you please check if my commit fixup is ok, first time I played with that. Thx

Reply:
> Could you please check if my commit fixup is ok, first time I played with that. Thx

Looks good.

> To be honest, I don't know why I did put those... I just played with lwip example, and try it to make it work. I know it's not so pro, but it worked. I put some comment though.

Then maybe, if its not too much hassle for you, check if they are really needed or can be removed.

Reply:
I can do that for sure, but later, have things to do now. Thx again :)

Reply:
@javierfileiv The following code must then be embedded in something like `#ifdef SOCK_HAS_IPV6 ... #else ... #endif`: https://github.com/RIOT-OS/RIOT/blob/25a49d23467aefa681b692057b0bb92707885856/pkg/paho-mqtt/contrib/riot_iface.c#L113